doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/baseboard-heaters.tex

```diff
@@ -1,24 +1,28 @@
 \section{Baseboard Heaters }\label{baseboard-heaters}
 
-\subsection{Hot Water Baseboard Heater with Only Convection}\label{hot-water-baseboard-heater-with-only-convection}
+There are five different baseboard models in EnergyPlus.  Two of these models are convection only while three of these models have both radiative and convective heat transfer to the zone.  The hot water and electric baseboard models with only convection provide a simple convective heat gain to the space via the baseboard unit.  The models for these convection only units are fairly simple and simply provide convective heating to the space in an attempt to meet whatever heating load is present.  The convection only models do not require iteration or resimulation of the surface heat balances because it is assumed that there is no radiation given off by the unit.
```

Comment:
fairly simple and ~~simply~~ provide

Reply:
@Myoldmopar Good catch--this should now be fixed.